include/bus.h

```diff
@@ -161,7 +161,7 @@ static inline void bus_intr_teardown(device_t *dev, intr_handler_t *handler) {
   BUS_DRIVER(dev)->bus.intr_teardown(dev, handler);
 }
 
-/*! \brief Allocates resource for a device of size \a size between
+/*! \brief Allocates for a device resource of size \a size between
```

Comment:
IMO "Allocates resource of a type \a type and size \a size between \a start and \a end, for a device \a dev" sounds better.

Reply:
Agreed!